river/anomaly/base.py

```diff
@@ -42,3 +43,20 @@ def score_one(self, x: dict) -> float:
         normal observation.
 
         """
+
+    def predict_proba_one(self, x: dict) -> typing.Dict[base.typing.ClfTarget, float]:
+        """Predict the probability of each label for a dictionary of features `x`.
+
+                Parameters
+                ----------
+                x
+                    A dictionary of features.
+
+                Returns
+                -------
+                A dictionary that associates a probability which each label.
+
+                """
+
+        p = self.score_one(x=x)
+        return {True: p, False: 1.0 - p}
```

Comment:
I'm not comfortable with this. The output of `score_one` is not a probability, it's a score. Even if it's between 0 and 1, it's not a probability. I would rather stick with the notion of `score_one` and make it work with the rest of River.

Reply:
You're right. I removed it.